draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -484,18 +490,16 @@ authorized(AS x, AS y) =  / Else, "Provider+" if the U-SPAS entry
       <section title="Incongruence in IPv4 and IPv6 Connectivity">
         <t>
           The U-SPAS contain the union of Providers for a CAS for both IPv4 and IPv6 unicast connectivity.
-          This design choice consequently means that if a customer-provider relationship exists for one address family but doesn't exist for the other address family, AS_PATH verification outcomes for the latter AFI will be as permissive as verification outcomes for the former AFI.
-          That is believed to be a reasonable compromise as both the ASPA registration and verification processes are simplified, and no false positive outcomes are yielded (e.g. inadvertent Invalid evaluation).
+          This design choice consequently means that if a customer-provider relationship exists for one address family but does not exist for the other address family, AS_PATH verification outcomes for the latter AFI will be as permissive as verification outcomes for the former AFI.
+          That is believed to be a reasonable compromise as both the ASPA registration and verification processes are simplified, and no false positive outcomes are yielded (e.g., inadvertent Invalid evaluation).
         </t>
       </section>
 
-      <section title="Provider as Trusted Point">
-        <t>
-          Since an upstream provider becomes a trusted point, in theory, it might be able to propagate some instances of hijacked prefixes with forged-origin or forged-path-segment or even routes with manipulated AS_PATHs, and such attacks might go undetected by its customers or upper providers.
-        </t>
+      <section title="Attack by a Provider">
         <t>
-          While such attacks may happen, it does not seem to be a realistic scenario.
-          Normally a customer and their transit provider would have a signed agreement, and a policy violation (of the above kind) should have legal consequences or the customer can just drop the relationship with such a provider and remove the corresponding ASPA record.
+          An upstream provider might be able to propagate some instances of hijacked prefixes with forged-origin or forged-path-segment to a customer. While such attacks may happen, it does not seem to be a realistic scenario. Normally a customer and their transit provider would have a signed agreement, and a policy violation (of the above-mentioned kind) should have legal consequences or the customer can just drop the relationship with such a provider and remove the corresponding ASPA record.
+
```

Comment:
Why do you want to change this wording?

Reply:
@mitradir 
v-19 text says "...such attacks might go undetected by its customers or upper providers." But the truth is that "hijacked prefixes with forged-origin or forged-path-segment" are detected by upper providers. Only customers cannot detect that. I feel that we do not need to mention other path manipulations here without saying what they are. Anyway, in the next section (9.3), we do talk about other path manipulations in the form of altering AS prepends.

I changed the section title (9.2) to "Attack by a Provider" from "Provider as Trusted Point" because it is not true that the provider is always considered as a trusted point by a customer in the draft procedures. Just that the customer is not able to detect some forms of attacks by the provider while it can detect other forms of attacks.

Reply:
But the truth is that "hijacked prefixes with forged-origin or forged-path-segment" are detected by upper providers.

They are not if a Provider prepends customer AS.